src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -87,11 +83,8 @@ public static String truncate(String inputString, int truncateLength) {
         if (inputString.length() <= truncateLength) {
             return inputString;
         }
-        String result = inputString;
-        if (inputString.length() > truncateLength) {
-            result = inputString.substring(0, truncateLength - 3) + "...";
-        }
-        return result;
+
+        return inputString.substring(0, truncateLength - 3) + "...";
```

Comment:
Does this work even if the inputString is shorter than `truncateLength`?

Reply:
Yes, but only if `inputString` is at least `truncateLength-3` characters long. That case is caught by the if-block before the return statement.